glide.yaml

```diff
@@ -1,5 +1,7 @@
 package: github.com/giantswarm/k8scloudconfig
 import:
+- package: github.com/giantswarm/apiextensions
+  version: clientset
```

Comment:
This is the branch we have to pin for the migration. After we run on CRDs I will switch this to master.